### DISCUSSION TITLE
ResourceDiscoveryGroups for Flex server

### DISCUSSION THREAD

**Discussion by LukaszSzalek-TomTom** [tomkerkhove/promitor#1882] [Q&A]
Hi What should be set as type for ResourceDiscoveryGroups in promitor discovery resource to have Azure Flexible Servers  tracked?
for single this works
resourceDiscoveryGroups: 
  - name: flexes_rdg
    type: PostgreSQL
    criteria:
      include:
        regions:
        - westeurope
        resourceGroups:
        - ******
What type is for Flexes? Is it ready on 0.6.0 release?

**Comment by tomkerkhove** [comment]
This has just been merged and is not released yet, but as per the docs:

> - Resource discovery will discover all types of servers and thus the used metrics should match all of the types.
>   - You can use tags to define which resources to include, if you need filtering capabilities.

https://docs.promitor.io/configuration/v2.x/metrics/postgresql#limitations

**Comment by LukaszSzalek-TomTom** [comment]
When do you plan to release new version of component?

**Reply by tomkerkhove** [reply]
You can see that on the milestones:
- https://github.com/tomkerkhove/promitor/milestone/30
- https://github.com/tomkerkhove/promitor/milestone/35

Planned work: https://github.com/users/tomkerkhove/projects/2/views/12

**Reply by tomkerkhove** [reply]
You can already test with the following images:
- https://github.com/tomkerkhove/promitor/pkgs/container/promitor-agent-resource-discovery-ci/11662400?tag=pr1876
- https://github.com/tomkerkhove/promitor/pkgs/container/promitor-agent-scraper-ci/11662419?tag=pr1876
